src/lib.rs

```diff
@@ -226,12 +226,45 @@
 
 use std::collections::HashMap;
 use std::env::VarError;
+use std::fmt::Debug;
 use std::str::FromStr;
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Arc, Condvar, Mutex, MutexGuard, RwLock, TryLockError};
 use std::time::{Duration, Instant};
 use std::{env, thread};
 
+type Callback = Box<dyn Fn() + Send + Sync>;
+
+#[derive(Clone)]
+struct SyncCallback {
+    callback: Arc<Callback>,
+}
+
+impl Debug for SyncCallback {
+    fn fmt(&self, _f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        Ok(())
```

Comment:
Do print something like `_f.write_str("SyncCallback(..)")` instead of doing nothing.

Reply:
Ok.